test/constructor.test.js

```diff
@@ -0,0 +1,23 @@
+const EventEmitter = require('events');
+const { getDefaultConfig } = require('./mocks');
+const RPReporter = require('./../lib/mochaReporter');
+
+describe('constructor', function() {
+  test('should create RP client instance with passed options', function() {
+    const options = getDefaultConfig();
+    const runner = new EventEmitter();
+    const reporter = new RPReporter(runner, options);
+    expect(reporter.rpClient).toBeDefined();
```

Comment:
I think it's a good practice to add separation between your test structure.
Like in the example below
https://github.com/goldbergyoni/javascript-testing-best-practices#-%EF%B8%8F-12-structure-tests-by-the-aaa-pattern